### PR TITLE
Pregel various fixes

### PR DESCRIPTION
**Pull request by graetzer**
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr-linux/5757/

### CODE REVIEW

File: arangod/Pregel/GraphStore.cpp [hunk 1]
```diff
@@ -100,7 +99,7 @@ std::map<CollectionID, std::vector<VertexShardInfo>> GraphStore<V, E>::_allocate
 
   std::map<CollectionID, std::vector<VertexShardInfo>> result;
 
-  LOG_TOPIC(DEBUG, Logger::PREGEL) << "Allocating memory";
+  LOG_TOPIC(INFO, Logger::PREGEL) << "Allocating memory";
```
Comment (jsteemann) [inline arangod/Pregel/GraphStore.cpp]: I think this should stay at `DEBUG`?
```suggestion
  LOG_TOPIC(DEBUG, Logger::PREGEL) << "Allocating memory";
```

File: arangod/Pregel/GraphStore.cpp [hunk 2]
```diff
@@ -174,6 +173,7 @@ std::map<CollectionID, std::vector<VertexShardInfo>> GraphStore<V, E>::_allocate
                        eCount * _graphFormat->estimatedEdgeSize();
   if (!_config->lazyLoading() &&
       (_config->useMemoryMaps() || requiredMem > totalMemory / 2)) {
+    LOG_TOPIC(INFO, Logger::PREGEL) << "Using memory mapped storage";
```
Comment (jsteemann) [inline arangod/Pregel/GraphStore.cpp]: same
```suggestion
    LOG_TOPIC(DEBUG, Logger::PREGEL) << "Using memory mapped storage";
```

File: arangod/Pregel/GraphStore.cpp [hunk 3]
```diff
@@ -608,9 +604,10 @@ void GraphStore<V, E>::_storeVertices(std::vector<ShardID> const& globalShards,
 
 template <typename V, typename E>
 void GraphStore<V, E>::storeResults(WorkerConfig* config,
-                                    std::function<void(bool)> const& cb) {
-  _config = config;
+                                    std::function<void()> cb) {
+  LOG_TOPIC(INFO, Logger::PREGEL) << "Storing vertex data";
```
Comment (jsteemann) [inline arangod/Pregel/GraphStore.cpp]: same
```suggestion
  LOG_TOPIC(DEBUG, Logger::PREGEL) << "Storing vertex data";
```

**Comment by graetzer** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr-linux/5779/

**Comment by graetzer** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr-linux/5788/

File: arangod/Cache/CacheManagerFeature.cpp [hunk 4]
```diff
@@ -84,7 +84,7 @@ void CacheManagerFeature::validateOptions(std::shared_ptr<options::ProgramOption
     FATAL_ERROR_EXIT();
   }
 
-  if (_cacheSize < (CacheManagerFeature::minRebalancingInterval)) {
+  if (_rebalancingInterval < (CacheManagerFeature::minRebalancingInterval)) {
```
Comment (jsteemann) [inline arangod/Cache/CacheManagerFeature.cpp]: :+1:

**Comment by jsteemann** [comment]
https://jenkins01.arangodb.biz/view/PR/job/arangodb-matrix-pr/4295/

**Comment by jsteemann** [comment]
https://jenkins01.arangodb.biz/view/PR/job/arangodb-matrix-pr/4316/

**Comment by jsteemann** [comment]
https://jenkins01.arangodb.biz/view/PR/job/arangodb-matrix-pr/4323/

**Pull request merged**